### DISCUSSION TITLE
Is the change in font styles to the expected behavior?

### DISCUSSION THREAD

**Discussion by courtnicole** [incluud/accessible-astro-starter#99] [Q&A]
If not, what's a workaround to revert to the previous styled fonts? It's very visible in the hero fonts, but it also impacts other components.  Based on ~15 minutes of testing, it looks like it could be related to the interplay between Sass and tailwind?
I noticed this on my webpage today (https://huttonpospick.com/) without changing any of the related components. I can also see that the same has happened on the live preview page. 

If this is the intended behavior, is there a workaround? I tried forcing font sizes and weights via tailwind and could not restore the old functionality. 

### Previous: 
![image](https://github.com/user-attachments/assets/cf5996cb-0693-49ca-8ca7-14c7ca407f8a)

### Current: 
![image](https://github.com/user-attachments/assets/8fb8742d-138b-4d58-a8fc-e6c6962651fc)

**Comment by markteekman** [comment]
Hey @courtnicole!

Hmmm, interesting, I haven't seen this one before 🤔 And strangely enough, when I visit your website (which looks cool by the way!), or the Accessible Astro demo for that matter, it seems to be the right font:

<img width="1340" alt="Scherm­afbeelding 2024-10-02 om 19 32 12" src="https://github.com/user-attachments/assets/c73d54ee-ebad-4281-8650-da6ccd4b0eef">

What OS and browser are you viewing the websites in?

I really like your three.js effect on the bottom of the website 😃 Just remember that visitors should be able to pause it to keep your website fully accessible 🙂 

Regards,
Mark
